specs/altair/beacon-chain.md

```diff
@@ -392,7 +390,7 @@ def get_inactivity_penalty_deltas(state: BeaconState) -> Tuple[Sequence[Gwei], S
 
 #### Modified `slash_validator`
 
-*Note*: The function `slash_validator` is modified to use `MIN_SLASHING_PENALTY_QUOTIENT_ALTAIR` 
+*Note*: The function `slash_validator` is modified to use `MIN_SLASHING_PENALTY_QUOTIENT_ALTAIR`
```

Comment:
just formatting -- my editor found some whitespace

Reply:
Pedantic suggestion 😄:

```suggestion
*Note*: The function `slash_validator` is modified to use `MIN_SLASHING_PENALTY_QUOTIENT_ALTAIR`.
```

Reply:
oh actually this comment underflows and does terminate in a `.` on the next line -- i'll rollback this change